Install error with Makefile.MacOS

Env: 
- macOS Sierra with zsh
- I installed pass via "`brew install pass`"

When I "`sudo make install PREFIX=/usr/local`" (my pass installed via brew is here /usr/local/lib/password-store), I saw:
```
pass-otp git:(master) ✗ sudo make install PREFIX=/usr/local        
install: pass-otp.1 -> /usr/local/share/man/man1/pass-otp.1
install: otp.bash exists but is not a directory
make: *** [install] Error 71
```

After a small change like this, it succeed:
```
diff --git a/Makefile b/Makefile
index c3ad5b1..5654947 100644
--- a/Makefile
+++ b/Makefile
@@ -16,7 +16,7 @@ all:
 install:
        @install -v -d "$(DESTDIR)$(MANDIR)/man1" && install -m 0644 -v pass-$(PROG).1 "$(DESTDIR)$(MANDIR)/man1/pass-$(PROG).1"
        @install -v -d "$(DESTDIR)$(SYSTEM_EXTENSION_DIR)/"
-       @install -Dm0755 $(PROG).bash "$(DESTDIR)$(SYSTEM_EXTENSION_DIR)/$(PROG).bash"
+       @install -m0755 $(PROG).bash "$(DESTDIR)$(SYSTEM_EXTENSION_DIR)/$(PROG).bash"
        @echo
        @echo "pass-$(PROG) is installed succesfully"
        @echo
(END)
```

--- a/Makefile
+++ b/Makefile
@@ -16,7 +16,7 @@ all:
 install:
 	@install -v -d "$(DESTDIR)$(MANDIR)/man1" && install -m 0644 -v pass-$(PROG).1 "$(DESTDIR)$(MANDIR)/man1/pass-$(PROG).1"
 	@install -v -d "$(DESTDIR)$(SYSTEM_EXTENSION_DIR)/"
-	@install -Dm0755 $(PROG).bash "$(DESTDIR)$(SYSTEM_EXTENSION_DIR)/$(PROG).bash"
+	@install -v -m0755 $(PROG).bash "$(DESTDIR)$(SYSTEM_EXTENSION_DIR)/$(PROG).bash"
 	@echo
 	@echo "pass-$(PROG) is installed succesfully"
 	@echo
@@ -24,8 +24,7 @@ install:
 uninstall:
 	@rm -vrf \
 		"$(DESTDIR)$(SYSTEM_EXTENSION_DIR)/$(PROG).bash" \
-		"$(DESTDIR)$(IMPORTERS_DIR)/" \
-		"$(DESTDIR)$(MANDIR)/man1/pass-$(PROG).1" \
+		"$(DESTDIR)$(MANDIR)/man1/pass-$(PROG).1"
 
 test:
 	make -C tests

